src/typing_extensions.py

```diff
@@ -969,7 +987,7 @@ def __subclasscheck__(cls, other):
     _TypedDict = type.__new__(_TypedDictMeta, 'TypedDict', (), {})
 
     @_ensure_subclassable(lambda bases: (_TypedDict,))
-    def TypedDict(typename, fields=_marker, /, *, total=True, **kwargs):
+    def TypedDict(typename, fields=_marker, /, *, total=True, closed=False, **kwargs):
```

Comment:
For backward compatibility, if `closed` exists but is not True/False, add it to kwargs. Add a test case like this:

```
TD = TypedDict("TD", closed=int)
assert TD.__closed__ is False
assert TD.__annotations__ = {"closed": int}
```